common/thorhelper/thorcommon.hpp

```diff
@@ -212,6 +212,7 @@ interface ICompressor;
 extern THORHELPER_API IExtRowWriter *createRowWriter(IFile *file, IRowInterfaces *rowIf, unsigned flags=DEFAULT_RWFLAGS, ICompressor *compressor=NULL, size32_t compressorBlkSz=0);
 extern THORHELPER_API IExtRowWriter *createRowWriter(IFileIO *iFileIO, IRowInterfaces *rowIf, unsigned flags=DEFAULT_RWFLAGS, ICompressor *compressor=nullptr, size32_t compressorBlkSz=0);
 extern THORHELPER_API IExtRowWriter *createRowWriter(IFileIOStream *strm, IRowInterfaces *rowIf, unsigned flags=DEFAULT_RWFLAGS); // strm should be unbuffered
+extern THORHELPER_API IExtRowWriter *createRowWriter(IFileIOStream *strm, IRowInterfaces *rowIf, IPropertyTree * options, unsigned flags = DEFAULT_RWFLAGS);
```

Comment:
I am not sure about this part - I think this functionality is recreated fresh in thorwrite.cpp.  Probably modelling a similar structure to thorread.cpp.

Reply:
Code now throws an exception instead of trying to leverage a new createRowWriter() and supporting classes. We'll deal with that later.